build.cake

```diff
@@ -177,6 +180,42 @@ Task("TestAwsAuthentication")
             );
         });
 
+Task("TestPlainAuthentication")
```

Comment:
These are adding targets for GSSAPI tests and LDAP tests.

Reply:
They were not being run before as part of a cake target.

Reply:
Shouldn't running these tests be part of some other ticket?

This doesn't seem related to running on Linux.